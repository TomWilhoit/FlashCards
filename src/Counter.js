import React, { Component } from 'react';

class Counter extends Component {
  render() {
    return (
      <div className="Counter">
        <h2>Counter</h2>
      </div>
    );
  }
}

export default Counter