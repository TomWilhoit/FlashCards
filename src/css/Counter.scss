.counter {
  justify-content: right;
  font-size: 30px;
}