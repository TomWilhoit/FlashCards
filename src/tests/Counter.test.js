import React from 'react';
import Counter from '../Counter';
import { shallow } from 'enzyme';

describe('Counter', () => {
  
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Counter/>
      )
    });
    it('should match snapshot when all data is passed correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  })
