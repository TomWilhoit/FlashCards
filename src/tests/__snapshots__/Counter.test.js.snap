// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter should match snapshot when all data is passed correctly 1`] = `
<div
  className="Counter"
>
  <h2>
    Question 
    NaN
    /40
  </h2>
</div>
`;
